api/v0/runtime/runtime.proto

```diff
@@ -299,7 +299,7 @@ message CpuController {
   // In one period (1_000_000), how much can the tasks run.
   //
   // * Minimum: 0
-  optional int64 max = 2;
+  optional int64 limit = 2;
```

Comment:
i know you decided to rename this, but if the point is to stay close to the cgroups v2 API then this should absolutely be `max`.  but it also should take the value "max", not just an integer.

Reply:
Good point re naming.

For accepted values, "max" is the default so equivalent of not settings limit. Thoughts?

Reply:
you can also set it explicitly to "max".  i think it's ok to document that as the default as i can't see a user wanting to explicitly set it to unlimited...

Reply:
I'll improve the documentation. We could later introduce -1 as asking to explicitly set max, without breaking the api, but that wouldn't match the underlying api.

Reply:
Another option would be to later introduce a oneof, but I understand we are trying to stay away from oneofs.

Unless we introduce the ability to adjust the resources of a cell, I don't think an explicit max will ever be needed and even then, we can change the implementation to always set "max" when the field is `None` instead of relying on the default